### ISSUE TITLE
FR: Setting to disable audio forwarding

### ISSUE THREAD

**Issue by tyler36** [Opticos/GWSL-Source#76]
# Problem
When running GWSL, Windows sounds don't work. IE. New mail / chat notifications on Win10 are muted (no sound).

I run E2E testing (Cypress) via a Docker image in WSL2. I have been using vcxsrv and don't have a need for WSL2 or Docker image sound.

Therefore, I would like the ability to redirect display but without the GWSL Pulse Serve. I search the documentation and Issues for something, but without lucky. 

# Options
I was hoping for a config setting that I could enable/disable.  I assume the feature would be enable by default. Maybe something like ...

```json
 "general": {
  "pulse_audio": true
...
 },
```

Any thoughts?

**Comment by Pololot64** [comment]
I actually wrote this feature today and it will be included in the next release. No ETA yet but it should be soon :)

**Comment by tyler36** [comment]
I'm happy to give it a test run if you want so feedback.

Regardless, thank you for the update. 👍 

If the issue is still open, I'll try and post feedback after I give it a whirl, whenever that my be.

**Comment by Pololot64** [comment]
Sounds good :)

**Comment by tyler36** [comment]
Just updated to `1.3.9` from Microsoft store.

I selected `Configure GWSL` and saw there was a setting under general.
```json
 "general": {
  "pulseaudio": true,
```

I changed it to `false` and restarted the app. 
Windows retained sound control while GWSL worked as expected. 

Thank you for the app and implementing the feature.  It functions exactly as I need and expected. 👍

The only real feedback I can give is maybe adding a checkbox in the context menu, perhaps under "shared clipboard"? I currently have no need to toggle this on / off and don't see the need but perhaps it might be helpful for others.

Since the feature works exactly as requested, I'll close the ticket. Thanks again!
